src/Microsoft.PowerShell.Commands.Management/commands/management/Hotfix.cs

```diff
@@ -185,7 +185,7 @@ private bool FilterMatch(ManagementObject obj)
                 foreach (string desc in Description)
                 {
                     WildcardPattern wildcardpattern = WildcardPattern.Get(desc, WildcardOptions.IgnoreCase);
-                    if (wildcardpattern.IsMatch((string)obj["Description"]))
+                    if (wildcardpattern.IsMatch((string)obj[nameof(Description)]))
```

Comment:
Here Description is a property name of ManagementObject.
Or can it be:
```suggestion
                    if (wildcardpattern.IsMatch((string)obj[nameof(obj.Description)]))
```

Also what about "InstalledBy" in the file?

I suggest to revert such non-obvious changes here and below because we can break something tricky.